docs/testing/troubleshooting.md

```diff
@@ -86,6 +86,12 @@ The following steps should help troubleshoot any problems related to the **Autom
 > [!NOTE]
 > There is a known issue that prevents scripts stored in SharePoint from always appearing in the recently used list. This occurs when your admin turns off Exchange Web Services (EWS). Your SharePoint-based scripts are still accessible and usable through the file dialog.
 
+## Problems scheduling a script
+
+Scheduling a script requires a business license of Microsoft 365. This is needed for the Power Automate flow that powers the scheduled script. Errors from this flow are shown when there are problems scheduling. The following list shows common errors you might encounter.
+
+- **The access token expiry UTC time is earlier than current UTC time.**: Too much time has passed between signing into the service and scheduling the script. Reopen the script in the Code Editor and try to schedule again.
+
```

Comment:
Can we add that to edit the flow users need to go to the power automate website?

Reply:
Can do!